app/src/main/java/com/discordtime/gpts/maps/viewmodel/MapViewModel.kt

```diff
@@ -0,0 +1,53 @@
+package com.discordtime.gpts.maps.viewmodel
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import com.discordtime.gpts.R
+import com.discordtime.gpts.listplaces.model.Place
+import com.discordtime.gpts.repository.IPlaceRepository
+import com.discordtime.gpts.repository.PlaceRepository
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.BitmapDescriptor
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+class MapViewModel: ViewModel() {
+
+    private val repository: IPlaceRepository = PlaceRepository()
```

Comment:
Why don't we inject this? You defined a module for PlaceRepository, consider using it.